### PR TITLE
Created dashboard for publisher.

### PR DESCRIPTION
**Pull request by waheedahmed**
[ECOM-5174](https://openedx.atlassian.net/browse/ECOM-5174)

As a Publisher tool user, I need to be able to locate and click on the relevant navigation elements so that I can find my way around the tool.

Refer to the wireframe instructions below:
https://openedx.atlassian.net/wiki/display/ECOM/Steps+to+navigate+prototype

**Acceptance Criteria:**
1. Validate that from the dashboard, I can click on "Courses" on the left rail to get to the Courses page.
2. Validate that from the dashboard, I can click on "Studio Requests" on the left rail to get to the Studio Requests page.
3. Validate that from the dashboard, I can click on "Course Runs" on the left rail to get to the Course Run page.
4. Validate that the left rail is always visible, and the correct page is highlighted as I'm navigating 
   around the tool.

**Screenshots:**
screencapture-localhost-8004-publisher-course_runs-1474981398069

screencapture-localhost-8004-publisher-course_runs-1474981800101

screencapture-localhost-8004-publisher-course_runs-18-1474982000315

### CODE REVIEW

**Comment by waheedahmed** [comment]
@bderusha @awais786 Please review this.

**Comment by waheedahmed** [comment]
@bderusha We don't have courses list page right, I think we need to create a story for this. For now I used new course url instead of course listing page.

File: course_discovery/templates/footer.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<footer class="publisher-footer border-top">
+    <p class="copy-base">
+        Copyright © 2016</time> edX.
```
Comment (bderusha) [inline course_discovery/templates/footer.html]: Can we make the year dynamic please

File: course_discovery/templates/footer.html [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+    </p>
+</footer>
```
Comment (bderusha) [inline course_discovery/templates/footer.html]: add newline

File: course_discovery/templates/header.html [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{% load staticfiles %}
+{% load i18n %}
+<header class="header">
+    <div class="layout layout-header">
+        <img src="{% static 'images/logo.png' %}">
+        <h1 class="hd-1 emphasized">{% trans "Publisher" %}</h1>
+    </div>
+</header>
```
Comment (bderusha) [inline course_discovery/templates/header.html]: add newline

**Review by bderusha — APPROVED** [review]
Looks good!

**Comment by waheedahmed** [comment]
@bderusha Please take another look, updated the PR and added new screenshots above.

**Comment by bderusha** [comment]
👍

File: course_discovery/settings/base.py [hunk 4]
```diff
@@ -177,6 +177,7 @@
                 'django.template.context_processors.tz',
                 'django.contrib.messages.context_processors.messages',
                 'course_discovery.apps.core.context_processors.core',
+                'django.core.context_processors.request',
```
Comment (awais786) [inline course_discovery/settings/base.py]: https://docs.djangoproject.com/en/1.10/releases/1.8/#django-core-context-processors
Built-in template context processors have been moved to django.template.context_processors

File: course_discovery/apps/publisher/views.py [hunk 5]
```diff
@@ -32,7 +32,18 @@ def get_queryset(self):
...
-        return [CourseRunWrapper(course_run) for course_run in course_runs]
+        return course_runs
+
+    def get_context_data(self, **kwargs):
+        context = super(CourseRunListView, self).get_context_data(**kwargs)
+        course_runs = context.get('object_list')
+        published_courseruns = course_runs.filter(
+            state__name=State.PUBLISHED
+        ).select_related('course').all().order_by('-state__modified')[:5]
+        unpublished_courseruns = course_runs.exclude(state__name=State.PUBLISHED)
```
Comment (awais786) [inline course_discovery/apps/publisher/views.py:43]: Please add a test case which test course runs, published_courseruns and unpublished_courseruns.

**Comment by awais786** [comment]
👍

**Pull request merged**